### PR TITLE
EES-1634 - fix removal of prerelease users

### PR DESCRIPTION
**Pull request by duncan-at-hiveit**
This PR:
- fixes the removal of prerelease users by Analysts

The problem was that the API call from the front end was not passing the email to the back end, and the back end was expecting to find a request body.

Fixed by adding the email as a request parameter to the delete call instead.

### CODE REVIEW

**Review by luke-h1 — APPROVED** [review]

File: src/explore-education-statistics-admin/src/services/preReleaseUserService.ts [hunk 1]
```diff
@@ -15,7 +15,7 @@ const preReleaseUserService = {
   },
   removeUser(releaseId: string, email: string): Promise<void> {
     return client.delete(`/release/${releaseId}/prerelease-users`, {
-      data: {
+      params: {
```
Comment (ntsim) [inline src/explore-education-statistics-admin/src/services/preReleaseUserService.ts:18]: Don't think query params is a great way of doing this as it leaves emails in your browsing history. I'm surprised that `data` wasn't working!

**Review by ntsim — CHANGES_REQUESTED** [review]

File: src/explore-education-statistics-admin/src/services/preReleaseUserService.ts (hunk 1 above)
Comment (duncan-at-hiveit) [reply src/explore-education-statistics-admin/src/services/preReleaseUserService.ts:18]: OK will take a look, ta

File: src/explore-education-statistics-admin/package.json [hunk 2]
```diff
@@ -41,7 +41,7 @@
     "@types/yup": "^0.28.3",
     "@typescript-eslint/eslint-plugin": "^4.4.0",
     "@typescript-eslint/parser": "^4.4.0",
-    "axios": "^0.20.0",
+    "axios": "0.21.0",
```
Comment (ntsim) [inline src/explore-education-statistics-admin/package.json:44]: Don't think we need an exact pin, would use `^0.21.0`

File: src/explore-education-statistics-common/package.json [hunk 3]
```diff
@@ -156,5 +156,8 @@
     "ancestorSeparator": " › ",
     "suiteName": "Common Jest tests",
     "outputName": "junit-common.xml"
+  },
+  "dependencies": {
+    "axios": "0.21.0"
```
Comment (ntsim) [inline src/explore-education-statistics-common/package.json:161]: This should be in `devDependencies`

File: src/explore-education-statistics-frontend/package.json [hunk 4]
```diff
@@ -4,7 +4,7 @@
   "private": true,
   "dependencies": {
     "applicationinsights": "^1.8.7",
-    "axios": "^0.20.0",
+    "axios": "0.21.0",
```
Comment (ntsim) [inline src/explore-education-statistics-frontend/package.json:7]: Same thing, don't think we need an exact version pin

**Review by ntsim — CHANGES_REQUESTED** [review]

**Review by ntsim — APPROVED** [review]

**Pull request merged**